src/components/App.css

```diff
@@ -0,0 +1,230 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.appConatiner {
+  background-color: var(--pale-grey);
+}
+
```

Comment:
## Improve the Design 
- [ ] Improve the page layout to match that of the design given. Currently, what we have is a page\s content leaning on the left side of the site and is not as responsive as expected. The styling should be **EXACTLY** the same as the one provided in the designs ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/))

|Required Design|Current Implemetation|
|:---:|:---:|
|![bookstore-design](https://user-images.githubusercontent.com/22345961/105754712-7389b780-5f5b-11eb-83d7-8d910a2ef3d0.png)|![bookstore-implementation](https://user-images.githubusercontent.com/22345961/105754745-7be1f280-5f5b-11eb-9f80-5a3180f0a89b.png)|